src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -57,18 +58,6 @@ private static IEnumerable<string> GetClassesForType(Type type)
 			}
 		}
 
-		private void CapturePointerNative(Pointer pointer)
-		{
-			var command = "Uno.UI.WindowManager.current.setPointerCapture(" + HtmlId + ", " + pointer.PointerId + ");";
```

Comment:
We are loosing the native pointer capture capability?

Reply:
No it has been moved in [UIElement.Pointers.wasm.cs](https://github.com/unoplatform/uno/pull/1256/files#diff-a1ec1a13fbf77968165020e117b951faR199)